plugins/pay.c

```diff
@@ -103,6 +103,9 @@ struct pay_command {
 	/* Any remaining routehints to try. */
 	struct route_info **routehints;
 
+	/* Disable the use of shadow route ? */
+	double use_shadow;
+
```

Comment:
Can we make this dev-only please?  I don't want normal people to disable shadow routing, it's a terrible idea.

Reply:
Will do